src/util/Constants.js

```diff
@@ -1141,6 +1141,12 @@ exports.APIErrors = {
   INVALID_JSON: 50109,
   CANNOT_MIX_SUBSCRIPTION_AND_NON_SUBSCRIPTION_ROLES_FOR_EMOJI: 50144,
   CANNOT_CONVERT_PREMIUM_EMOJI_TO_NORMAL_EMOJI: 50145,
+  VOICE_MESSAGES_DO_NOT_SUPPORT_ADDITIONAL_CONTENT: 50159,
+  VOICE_MESSAGES_MUST_HAVE_A_SINGLE_AUDIO_ATTACHMENT: 50160,
+  VOICE_MESSAGES_MUST_HAVE_SUPPORTING_METADATA: 50161,
+  VOICE_MESSAGES_CANNOT_BE_EDITED: 50162,
+  CANNOT_DELETE_GUILD_SUBSCRIPTION_INTEGRATION: 50163,
```

Comment:
This is not part of the upstream PR nor the docs PR.

Reply:
It is part of discord api types pr

Reply:
It's not part of that pull request. It was part of https://github.com/discordjs/discord-api-types/pull/725. You should remove it.

Reply:
Oh I copied too much
Okey, I will delete it